### DISCUSSION TITLE
Therminol D12 Properties

### DISCUSSION THREAD

**Discussion by Princerivo** [DanWBR/dwsim#558] [General]
Hi,
   I am wondering if anyone got the properties for the Therminol D12 heat transfer fluid. I am struggling to find it anywhere.

Regards,
M

**Comment by DanWBR** [comment]
did you try coolprop incompressible fluids property package?

**Reply by Princerivo** [reply]
Hi Dan,
     If it is there do you know how to export it to DWSIM. Also, is there a way like specifying the fluid properties, Cp, density...etc. instead?

Regards,
M.
